clarifai/cli/deployment.py

```diff
@@ -0,0 +1,49 @@
+import click
+from clarifai.cli.base import cli
+from clarifai.client.nodepool import Nodepool
+from clarifai.utils.cli import display_co_resources
+
+
+@cli.group()
+@click.option(
+    '--nodepool_id', required=True, help='Nodepool ID for the Nodepool to interact with.')
```

Comment:
same comment as nodepool and compute cluster

Reply:
https://github.com/Clarifai/clarifai-python/pull/436#discussion_r1827566302